serve/mlc_serve/engine/sampling_params.py

```diff
@@ -104,8 +117,16 @@ def _verify_args(self) -> None:
                     raise ValueError(
                         f"logit bias must be in [-100, 100], got {bias} for token {token}."
                     )
+                if not 1 <= token <= self.vocab_size:
+                    raise ValueError(f"index must be in [1, vocab_size]")
```

Comment:
what index

Reply:
Changed to token id. Thanks for the catch!